homeassistant/components/velbus/__init__.py

```diff
@@ -42,68 +53,88 @@ async def async_setup(hass, config):
     return True
 
 
+async def velbus_connect_task(
+    controller: Velbus, hass: HomeAssistant, entry_id: str
+) -> None:
+    """Task to offload the long running connect."""
+    # create notification
+    persistent_notification.async_create(
```

Comment:
Let's drop the persistent notification. We can add this to the user docs instead.